MapboxDirections/MBDirections.swift

```diff
@@ -10,18 +10,18 @@ let defaultApiEndPointURLString = Bundle.main.object(forInfoDictionaryKey: "MGLM
 /// The user agent string for any HTTP requests performed directly within this library.
 let userAgent: String = {
     var components: [String] = []
-    
```

Comment:
FYI, it looks like your editor is configured to automatically remove indentation from otherwise blank lines. I’m officially indifferent to whitespace changes, but you might find it easier to navigate the diff if it includes only documentation changes.